### PR TITLE
Generate function documentation in README.md

### PR DESCRIPTION
**Pull request by palortoff**
Function documentation in the README.md is now generated from the code documentation in index.js.

This way we avoid additional maintenance effort and duplication.

### CODE REVIEW

**Comment by edwellbrook** [comment]
Could you rebase on `new-tvdb-api` branch and I'll take a look :)

**Comment by palortoff** [comment]
@edwellbrook: rebased onto new-tvdb-api

File: scripts/updateReadme.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    if (comment.examples.length === 0){
+        throw new Error(`example code missing for ${comment.name} doc (index.js:${comment.comment.begin})`);
+    }
+
+    comment.exampleCode = comment.examples[0].block.replace(/\\/g, '');
+
+    return functionDocTemplate(comment);
+}
+
+updateReadme();
```
Comment (edwellbrook) [inline scripts/updateReadme.js]: New line, please

File: scripts/updateReadme.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+let parseComments = require('parse-comments');
```
Comment (edwellbrook) [inline scripts/updateReadme.js]: Can we have these and other variables that do not change be declared with `const`

File: package.json [hunk 3]
```diff
@@ -29,11 +29,13 @@
...
     "eslint-plugin-mocha": "4.5.1",
-    "mocha": "3.1.0"
+    "mocha": "3.1.0",
+    "parse-comments": "0.4.3"
   },
   "scripts": {
     "test": "mocha test",
-    "eslint": "eslint ."
+    "eslint": "eslint .",
+    "updateReadme": "node ./scripts/updateReadme"
```
Comment (edwellbrook) [inline package.json]: Personal preference here would be `"update-readme": "node ./scripts/update-readme"`

**Comment by palortoff** [comment]
@edwellbrook: all done

**Pull request merged**